src/ConstrIndex.purs

```diff
@@ -0,0 +1,39 @@
+module ConstrIndex
```

Comment:
Could we add some docs to the module? I think the relation between this module and its definitions and `From`/`ToData` might not be entirely clear

Reply:
Done!